snap/README.md

```diff
@@ -260,11 +248,33 @@ sudo snap set edgexfoundry env.security-proxy.user=user01,USER_ID,ES256
 sudo snap set edgexfoundry env.security-proxy.public-key="$(cat public.pem)"
 ```
 
-The second step is then to generate a token using the user ID which you specified:
+Alternatively, you can create the user using the secrets-config command. You need to provide the following:
+
+- The username
+- The public key
+- The API Gateway Admin JWT token
+- (optionally) ID. This is a unique string identifying the credential. It will be required in the next step to
+create the JWT token. If you don't specify it,
+then an autogenerated one will be output by the secrets-config command
+```bash
+
+# get API Gateway/Kong token
+JWT_FILE=/var/snap/edgexfoundry/current/secrets/security-proxy-setup/kong-admin-jwt
+JWT=`sudo cat ${JWT_FILE}`
+
+# use secrets-config to add user
+$ edgexfoundry.secrets-config proxy adduser --token-type jwt --user user01 --algorithm ES256 --public_key public.pem --id USER_ID --jwt ${JWT}
+```
+
+3. Finally, you need to generate a token using the user ID which you specified:
 
 ```bash
-$ edgexfoundry.secrets-config proxy jwt --algorithm ES256 --private_key private.pem --id USER_ID --expiration=1h
-TOKEN="copy manually from output of secrets-config command"
+# get token
+TOKEN=`edgexfoundry.secrets-config proxy jwt --algorithm ES256 --private_key private.pem --id USER_ID --expiration=1h`
+
+# Keep this token in a safe place for future reuse as the same token cannot be regenerated or recovered using the secret-config CLI
+echo $TOKEN
```

Comment:
echo $TOKEN
level=ERROR ts=2021-11-11T12:11:24.430840404Z app=secrets-config source=bootstraphandler.go:76 msg="Could read private key from file private.pem: open private.pem: permission denied"

sudo echo $TOKEN
level=ERROR ts=2021-11-11T12:11:24.430840404Z app=secrets-config source=bootstraphandler.go:76 msg="Could read private key from file private.pem: open private.pem: permission denied"

Reply:
Same as above. I was in the checkbox directory with previous key pairs.